web/src/types/error.type.ts

```diff
@@ -1,3 +1,6 @@
-export interface ErrorResponse {
+export interface ErrorResponse extends Error {
```

Comment:
Why extend error here?

Reply:
I meant for this error to be a standard error object. So im just extending it and requiring a code whenever you specify a cause now

Reply:
Ok, but then shouldn't we use `message` for the error string like `Error` does?